nemo/src/table_manager.rs

```diff
@@ -326,7 +326,8 @@ impl TableManager {
     pub fn table_serializer(&mut self, id: TableId) -> Result<impl TrieSerializer + '_, Error> {
         let _ = self
             .database
-            .get_trie_or_load(id, &ColumnOrder::default())?;
+            .get_trie_or_load(id, &ColumnOrder::default())
+            .map_err(|e| Error::PhysicalError(e.into()))?;
```

Comment:
Can't we have an `impl From<ReadingError> for nemo::error::Error` that does the wrapping inside `Error::PhysicalError` for us?